SystemTextJsonPatch/Internal/PropertyProxyCache.cs

```diff
@@ -49,11 +51,20 @@ internal static class PropertyProxyCache
 			{
 				if (string.Equals(propertyInfo.Name, propName, StringComparison.OrdinalIgnoreCase))
 				{
+					EnsureAccessToProperty(propertyInfo);
 					return new PropertyProxy(propertyInfo);
 				}
 			}
 
 			return null;
 		}
+
+		private static void EnsureAccessToProperty(PropertyInfo propertyInfo)
+		{
+			if (propertyInfo.GetCustomAttribute(typeof(DenyPatchAttribute), true) != null)
```

Comment:
Wasn't there also generic overload `propertyInfo.GetCustomAttribute<DenyPatchAttribute>(...)` `?

Reply:
Its not a big deal anyway

Reply:
Yes there is, but i don't think it exists in `netstandard2.0`.